audify/cli.py

```
@@ -2,36 +2,532 @@
+            # Generate the audiobook
+            output_path = dir_creator.synthesize()
+
+            logger.info("\n" + "=" * terminal_width)
+            logger.info("Directory audiobook creation complete!")
+            logger.info(f"Output directory: {output_path}")
+            logger.info("=" * terminal_width)
+
+        except KeyboardInterrupt:
+            logger.info("\n\nDirectory audiobook creation cancelled by user.")
+            return
+        except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)